src/templates/users/new_user.html

```diff
@@ -0,0 +1,24 @@
+
+  {% extends 'layout.html'%}
+  {% block content %}
+  
+  
+  <main>
+    <h1>Create a new User</h1>
+  <form action="/users/create" method="post">
+    <div class="mb-3">
+    <label class="form-label" for="name">Name:</label><br>
+    <input class="form-control" type="text" id="name" name="name"><br>
+    <label class="form-label" for="email">Email:</label><br>
+    <input class="form-control" type="email" id="email" name="email"><br>
+    <label class="form-label" for="password">Password:</label><br>
+    <input class="form-control" type="password" id="password" name="password"><br>
+    <label class="form-label" for="datetime">Date:</label><br>
+    <input class="form-control" type="date" id="datetime" name="datetime"><br>
```

Comment:
Essa data serve para que?
Se for a data de criação do user, vc deve obter ela no servidor e não do usuário.